packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpApi.ts

```diff
@@ -183,10 +204,14 @@ export class FtpUssApi implements ZoweExplorerApi.IUss {
         const connection = await this.ftpClient(this.checkedProfile());
         if (connection && connection.client) {
             if (type === "directory") {
-                await connection.client.mkdir(ussPath);
-            } else if (type === "file") {
+                await UssUtils.makeDirectory(connection, ussPath);
+            } else if (type === "File" || type === "file") {
                 const content = Buffer.from(CoreUtils.addCarriageReturns(""));
-                await connection.uploadDataset(content, ussPath, "ascii");
+                const transferOptions = {
+                    transferType: "ascii",
```

Comment:
Can use `TRANSFER_TYPE_ASCII` here

Reply:
I will change it to TRANSFER_TYPE_ASCII. Thanks!